android/app/src/main/java/com/mattermost/rnbeta/MainActivity.java

```diff
@@ -4,11 +4,28 @@
 import androidx.annotation.Nullable;
 
 import com.reactnativenavigation.NavigationActivity;
+import android.view.KeyEvent;
+import com.github.emilioicai.hwkeyboardevent.HWKeyboardEventModule;
 
 public class MainActivity extends NavigationActivity {
     @Override
     protected void onCreate(@Nullable Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.launch_screen);
     }
+
+    /*
+    https://mattermost.atlassian.net/browse/MM-10601
+    Required by react-native-hw-keyboard-event
+    (https://github.com/emilioicai/react-native-hw-keyboard-event)
```

Comment:
and with this @emilioicai just improved the RN ecosystem with a new library 🎉 

qq: @emilioicai do you want us to review the library too ?

Reply:
😄 Yeah, it would be great if you guys could take a look (I think @migbot already checked it).